Formula/lcm.rb

```diff
@@ -18,6 +18,11 @@ class Lcm < Formula
     end
   end
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    strategy :github_latest
```

The `stable` URL is a GitHub release asset, so this should use `GithubLatest` instead of checking the Git tags.